scripts/rpc-api-generation/index.ts

```diff
@@ -22,6 +22,14 @@ type ProtoServiceDefinition = {
   >
 }
 
+const deprecatedApiMethods: Record<string, string[]> = {
+  CommunicationsController: ['send'],
+  EngineApi: ['sendBatch', 'subscribe', 'unsubscribe'],
+  EthereumController: ['requirePayment', 'signMessage', 'convertMessageToObject', 'getUserAccount'],
+  Players: ['getPlayerData', 'getPlayersInScene', 'getConnectedPlayers'],
```

Comment:
should we add some text with the deprecated tag? i.e: "@deprecated use <this other api> instead"?